components/portal/flow/components/sidebar/builderMenu.tsx

```diff
@@ -61,21 +62,20 @@ const onNodeDescriptionChange = (event: React.ChangeEvent<{ value: unknown }>) =
 
   return (
     <div className="buildermenu">
-      <h3>Competentie</h3>
       <Box sx={{ p: 3 }}>
       <TextField
         fullWidth
-        label="Display Name"
+        label="Competentienaam"
```

Comment:
Waarom zijn de labels in het nederlands en niet engels? Wat voor invloed heeft dit of wat is de keuze hiervoor?

Reply:
Waarom zouden ze niet in het Nederlands moeten, wat voor invloed zou het hebben als dat het geval is?